target/linux/ath79/dts/qca9531_comfast_cf-wr610n.dts

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "comfast,cf-wr610n", "qca,qca9531";
+	model = "COMFAST CF-WR610N";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &wan;
+		led-failsafe = &failsafe;
+		led-upgrade = &failsafe;
+		label-mac-device = &eth1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wifi {
+			label = "green:wifi";
+			gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		failsafe:failsafe {
+			label = "red:failsafe";
+			gpios = <&gpio 2 GPIO_ACTIVE_HIGH>;
+		};
+
+		wan:wan {
```

Comment:
`led_wan: wan {`

or just

`wan {`

if you remove this from led-boot ..

Reply:
hi, can you review ?